ntbs-service/Helpers/NameFormattingHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Text.RegularExpressions;
+
+namespace ntbs_service.Helpers
+{
+    public static class NameFormattingHelper
+    {
+        public static string FormatDisplayName(string displayName)
```

Comment:
Is there any way that `displayName` could be `null`? That would cause problems if so.

Reply:
I'm not sure, so just added in a case to cover that in eef931e